cirq/iqm/iqm_device.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""IQM builds quantum computers based on superconducting transmon technology.
+For more information, see https://meetiqm.com.
+"""
+
+from typing import cast, Iterable, Set
+import cirq
+from cirq import devices, ops, protocols
+
+
+class Adonis(devices.Device):
+    """IQM's five-qubit superconducting device with pairwise connectivity.
+    The qubits are connected thus (recommend using fixed-width font for
+    viewing):
+         q0
+         |
+    q1 - q2 - q3
+         |
+         q4
```

Comment:
Instead of calling them q0, q1..., should they be 0_1, 1_0, 1_1, 1_2, and 2_0 since they are defined later as GridQubits?

Reply:
Implemented.